client/src/components/top.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <bias-form v-model="modelData.bias"/>
+</template>
+
+<script>
+import BiasForm from './BiasForm.vue';
+
+export default {
+  name: 'Top',
+  data: () => ({
+    modelData: {
+      bias: {
+        bias_model: null,
+        bias_params: null,
+      },
+    },
+  }),
+  watch: {
+    'modelData.bias': {
+      deep: true,
+      handler() {
+        console.log(this.modelData.bias);
+      },
+    },
+  },
+  components: {
+    BiasForm,
+  },
+};
+</script>
```

Comment:
What is this component for? Also why is it the only component without a capital first letter? 😛

Reply:
Just a temporary demo to show how state can be handled using v-model. Will be deleted in final version of this pull request.

Lowercase name is a reminder to delete.